Xamarin.Forms.Core/Items/ScrolledEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core.Items
+{
+	public class ScrolledEventArgs : EventArgs
```

Comment:
Should this extend the existing `ScrolledEventArgs` (which `CarouselView` spec seems to indicate)?